domains/sailesh.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Sailesh's Personal Website",
+  "owner": {
+    "username": "saileshbro",
+    "email": "saileshbro@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://saileshbro.github.io/saileshbro/",
```

@saileshbro 
CNAME needs to be saileshbro.github.io
you may want to add it as URL
```suggestion
    "URL": "https://saileshbro.github.io/saileshbro/",
```